EIPS/eip-2381.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 2381
+title: ENS support for ERC 721 TokenID
+author: Oisin Kyne (@OisinKyne) <oisin@kyne.eu>
+discussions-to: https://github.com/ethereum/EIPs/issues/2381
+status: Last Call
```

Comment:
This isn't even merged as a draft. It can't start out as last call.

Reply:
Sorry, I didn't realise they merge as draft, I figured the PR being merged was the last step. Will revert.